volumes/volume-manage.html.markerb

```diff
@@ -176,19 +178,34 @@ For options, refer to the [`fly machine clone` docs](/docs/flyctl/machine-clone/
 ## Destroy a volume
 
 <div class="warning icon">
-<b>Warning:</b> When you destroy a volume, you permanently delete all its data.
+<b>Warning:</b> When you destroy a volume, you permanently delete all its data. There is no undo. Make sure you've backed up any data you want to keep.
 </div>
 
-1. Run `fly volumes list` and copy the ID of volume to destroy.
+1. List volumes and find the ID for the one you want to destroy:
 
-2. Destroy the volume:
+```cmd
+fly volumes list
+```
+
+2. Destroy the volume using its ID:
 
 ```cmd
 fly volumes destroy <volume id>
 ```
 
 For options, refer to the [`fly volumes destroy` docs](/docs/flyctl/volumes-destroy/) or run `fly volumes destroy --help`.
 
+**Volumes attached to Machines**
+
+Volumes attached to Machines can’t be destroyed. You can check which Machine (if any) a volume is attached to by running `fly volumes list` First, stop and destroy the Machine, then the unattached volume can be deleted.
+
+If you’re replacing the Machine, use `fly scale count` or `fly machine clone` to launch a new one. This will create and mount a fresh volume based on your `fly.toml` config.
+
+To stop using volumes entirely, remove any `[mounts]` from `fly.toml`, redeploy to detach them, and then delete the volumes.
```

Comment:
nitpick: a couple of places say "delete" the volumes, do we want to standardize on calling it "destroy" the volumes? (also "deleted -> destroyed" a few lines above).

Reply:
Thanks! Fixed